reppy/parser.py

```diff
@@ -110,9 +110,12 @@ def __init__(self, url, status, content, expires):
         if status == 200:
             self.parse(content)
         elif status in (401, 403):
-            logger.warn('Access disallowed to site %s (%i)' % (
+            logger.warn('Access forbidden to site %s (%i)' % (
                 url, status))
-            self.parse('''User-agent: *\nDisallow: /''')
+            if disallow_forbidden:
+                self.parse('''User-agent: *\nDisallow: /''')
+            else:
+                self.parse('')
```

Comment:
Off hand, I don't recall why `self.parse('')` is called. My inclination is to leave it as it is and follow the example for the time being.